sql/pgwire_test.go

```diff
@@ -390,7 +390,11 @@ func TestPGPreparedQuery(t *testing.T) {
 				time.Date(2006, 7, 8, 0, 0, 0, 0, time.FixedZone("", 0)),
 			),
 		},
-
+		"INSERT INTO d.ts (a) VALUES ($1) RETURNING a": {
```

Comment:
`pq.ParseTimestamp` cannot parse the `time.RFC3339Nano` format. Not sure why that is being sent. Looking.